services/QuillLMS/app/services/classroom_students_diagnostic_progress_aggregator.rb

```diff
@@ -20,7 +20,7 @@ def run
   private def student_ids_with_completed_activity_sessions_counts
     ActivitySession
       .joins(:classroom_unit)
-      .where(activity_sessions: { user_id: student_ids, state: ActivitySession::STATE_FINISHED })
+      .where(activity_sessions: { user_id: student_ids, is_final_score: true })
```

Comment:
Had to change this so we don't wind up with "7 of 5 activities completed" when students replay activities.

Reply:
Nice job anticipating this edge case.